datasets.py

```diff
@@ -122,6 +122,36 @@ def load_labels(self) -> pd.DataFrame:
         return df
 
 
+class SudanBlueNileCEO2020(LabeledDataset):
+    def load_labels(self) -> pd.DataFrame:
+        SudanBlueNile_dir = raw_dir / "Sudan_Blue_Nile_CEO_2020"
+        df1 = pd.read_csv(
+            SudanBlueNile_dir
+            / "ceo-Sudan-Blue-Nile-Feb-2020---Feb-2021-(Set-1)-sample-data-2023-04-04.csv"
+        )
+        df2 = pd.read_csv(
+            SudanBlueNile_dir
+            / "ceo-Sudan-Blue-Nile-Feb-2020---Feb-2021-(Set-2)-sample-data-2023-04-04.csv"
+        )
+        df = pd.concat([df1, df2])
+        df[CLASS_PROB] = df["Does this pixel contain active cropland?"] == "Crop"
+        df[CLASS_PROB] = df[CLASS_PROB].astype(int)
+
+        df["num_labelers"] = 1
+        df = df.groupby([LON, LAT], as_index=False, sort=False).agg(
+            {
+                CLASS_PROB: "mean",
+                "num_labelers": "sum",
+                "plotid": join_unique,
+                "sampleid": join_unique,
+                "email": join_unique,
+            }
+        )
+        df[START], df[END] = date(2020, 1, 1), date(2021, 12, 31)
+        df[SUBSET] = train_val_test_split(df.index, 0.5, 0.5)
```

Comment:
There's 1500 points here, I think we can put less into the validation and test set, maybe `0.35, 0.35`

Reply:
Alright Ivan